### ISSUE TITLE
No canonical conversion from ℚ to ‾𝔽ₚ?

### ISSUE THREAD

**Issue by user202729** [sagemath/sage#38373]
### Steps To Reproduce

```python
p = 2**127 - 1
F = GF(p)

F.algebraic_closure()(F(5/2))  # okay
F.algebraic_closure()(5/2)  # error?
```

### Expected Behavior

The last line also works.

### Actual Behavior

The last line errors out.

Side note, the motivation is the following.

```python
p = 2**5-1
F = GF(p)
E = EllipticCurve(F, [0, 6, 0, 1, 0])
E.change_ring(F.algebraic_closure()).isogenies_prime_degree(5)  # errors?
```

### Checklist

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.
- [X] I have read the documentation and troubleshoot guide

**Comment by JohnCremona** [comment]
I agree that there should be a conversion from QQ to any field of characteristic p which works for rationals with non-negative valuation, i.e. with denominator prime to p.

**Comment by fchapoton** [comment]
The error happens when it tries to **coerce** the rational in the ground field.
